app/controllers/promos/PromoCampaignsController.scala

```diff
@@ -53,7 +53,11 @@ class PromoCampaignsController(
           dynamoPromoCampaigns
             .getAllPromoCampaigns(promoProduct)
             .map { promoCampaigns =>
-              Ok(noNulls(promoCampaigns.asJson))
+              val response = Map(
+                "promoCampaigns" -> promoCampaigns.asJson,
+                "userEmail" -> request.user.email.asJson
```

Comment:
is this needed in the campaigns response?

Reply:
Isn't that info returned by the Promos controller though?
This endpoint is returning the list of campaigns